File: modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/services/checker/trait/CheckerServiceHandle.scala
package de.thm.ii.fbs.services.checker.`trait`

import de.thm.ii.fbs.model.task.Task
import de.thm.ii.fbs.model.{CheckrunnerConfiguration, Submission}

trait CheckerServiceHandle {
  def handle(submission: Submission, checkerConfiguration: CheckrunnerConfiguration, task: Task, exitCode: Int, resultText: String, extInfo: String): Unit
}

File: modules/fbs-core/api/src/main/kotlin/de/thm/ii/fbs/services/v2/checker/trait/CheckerServiceHandle.kt
package de.thm.ii.fbs.services.v2.checker.trait

import de.thm.ii.fbs.model.v2.CheckrunnerConfiguration
import de.thm.ii.fbs.model.v2.Submission
import de.thm.ii.fbs.model.v2.Task

interface CheckerServiceHandle {
    fun handle(submission: Submission, checkerConfiguration: CheckrunnerConfiguration, task: Task, exitCode: Int, resultText: String, extInfo: String)
}